GA/repair.py

```
@@ -0,0 +1,556 @@
+
+        else:
+            x_LE_camber = np.zeros_like(x_LE_thickness)
+            x_TE_camber = np.zeros_like(x_TE_thickness)
+            y_LE_camber = np.zeros_like(y_LE_thickness)
+            y_TE_camber = np.zeros_like(y_TE_thickness)
+
+        return (x_LE_thickness, x_TE_thickness, x_LE_camber, x_TE_camber), \
+               (y_LE_thickness, y_TE_thickness, y_LE_camber, y_TE_camber)
+
+
+    def _enforce_one2one(self,
```

Pylint (3.3.7)

[refactor] Too many local variables (33/15)

(R0914)